### ISSUE TITLE
Invalid object name 'sys.sequences'.

### ISSUE THREAD

**Issue by James-DBA-Anderson** [SQLRockstar/BlogScripts#1]
Sequences were not implemented until SQL Server 2012. For this reason the SELECT from sys.sequences fails on pre 2012 instances. 

The strange thing is that [BOL](https://msdn.microsoft.com/en-us/library/ff877934.aspx) says this table has been present since 2008 but I am unable to see it on my 2008 and 2008R2 instances.

**Comment by SQLRockstar** [comment]
Yes, you are correct. We should either change the header to explain that it doesn't support pre-SQL2012 instances, or update the code to not try to select from sys.sequences unless it exists.

**Comment by echeypaz** [comment]
Hi, Are there any solution, for this issue?. Set propertie spring.jpa.hibernate.use-new-id-generator-mappings=false
or
org.hibernate.dialect.SQLServer2008Dialect not working for me.

**Comment by echeypaz** [comment]
I have found the solution.
**spring.jpa.database-platform=org.hibernate.dialect.SQLServer2008Dialect**
instance of
spring.jpa.hibernate.dialect=org.hibernate.dialect.SQLServer2008Dialect

**Comment by russellhoff** [comment]
In my case, I was making use of an SQL Server 2008 instance with 2012's dialect, that was the problem.
